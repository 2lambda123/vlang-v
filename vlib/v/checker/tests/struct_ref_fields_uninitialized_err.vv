struct ContainsRef {
	a &int = unsafe { nil }
	b &int
}

struct Outer {
	c ContainsRef
}

fn main() {
	o := Outer{}
	dump(o)
}
