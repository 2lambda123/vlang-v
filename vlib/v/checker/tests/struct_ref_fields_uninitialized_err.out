vlib/v/checker/tests/struct_ref_fields_uninitialized_err.vv:11:7: warning: reference field `Outer.c.b` must be initialized
    9 | 
   10 | fn main() {
   11 |     o := Outer{}
      |          ~~~~~~~
   12 |     dump(o)
   13 | }
